tiatoolbox/utils/misc.py

```diff
@@ -355,6 +358,53 @@ def contrast_enhancer(img, low_p=2, high_p=98):
     return np.uint8(img_out)
 
 
+def numpy_array_to_table(input_table):
```

Comment:
Should these be private function?

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/250/commits/4c2e04b86bfccdb876dd639de58b96f72885a20b